1-exercises/A-reading-list/script.js

```diff
@@ -1,5 +1,26 @@
 function readingList(books) {
-  // Write your code here...
+  const readingList = document.createElement(`ul`);
+
+  books.forEach((book) => {
+    const bookItem = document.createElement(`li`);
+
+    if (book.alreadyRead) bookItem.classList.add(`book-list-item__is-read`);
+    else bookItem.classList.add(`book-list-item__not-read`);
+
+    const bookInfo = document.createElement(`p`);
+    const bookImage = document.createElement(`img`);
+
+    const bookInfoText = `${book.title} by ${book.author}`;
+
+    bookInfo.innerText = bookInfoText;
+    bookImage.src = book.bookCoverImage;
+    bookImage.classList.add(`book-image-cover`);
+
+    bookItem.append(bookInfo, bookImage);
+    readingList.appendChild(bookItem);
+  });
+
+  content.appendChild(readingList);
 }
```

Comment:
This one is a big point. Unfortunately you haven't completed the requirements.

![Final Result Demo](https://github.com/CodeYourFuture/JavaScript-Core-2-Coursework-Week3/assets/91217473/48df35d6-50a4-4d14-bb1c-c8b21cedfbf7)

The fifth bullet point in the requirements for the exercise asked you to.

"Change the style of the book depending on whether you have read it (green) or not (red). Hint: try adding some CSS classes to styles.css for these styles".

Be sure to read and reread requirements, you will need to revisit them to keep the direction moving in the right direction and this is real world stuff. You will have requirements in the workplace to!